src/main/webapp/app/delete-dialog/delete-dialog-service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { DeleteDialogComponent } from 'app/delete-dialog/delete-dialog.component';
+
+/**
+ * Data that will be passed to the delete dialog component
+ */
+export class DeleteDialogData {
+    entityTitle: string;
+    deleteQuestion: string;
+    deleteConfirmationText?: string;
```

Comment:
The attributes should each be documented. E.g. if deleteConfirmationText is undefined, there will be no safety check.

Reply:
Added a short documentation here